apps/client/src/app/components/admin-market-data/create-asset-profile-dialog/create-asset-profile-dialog.html

```diff
@@ -6,21 +6,55 @@
 >
   <h1 i18n mat-dialog-title>Add Asset Profile</h1>
   <div class="flex-grow-1 py-3" mat-dialog-content>
-    <mat-form-field appearance="outline" class="w-100">
-      <mat-label i18n>Name, symbol or ISIN</mat-label>
-      <gf-symbol-autocomplete
-        formControlName="searchSymbol"
-        [includeIndices]="true"
-      />
-    </mat-form-field>
+    <div class="my-2">
+      <mat-radio-group name="radioOptions" [(ngModel)]="selectedOption">
+        <div class="flex">
+          <mat-radio-button
+            name="auto"
+            value="auto"
+            [checked]="selectedOption==='auto'"
+            (click)="onRadioChange('auto')"
+          ></mat-radio-button
+          ><label for="auto">Search</label>
```

Comment:
Remove `label`

```
<mat-radio-button>Search</mat-radio-button>
```

Reply:
<img width="947" alt="image" src="https://github.com/ghostfolio/ghostfolio/assets/77065548/6af5c8f5-ee2a-4639-a70c-b7dc874e925f">

Removing the `label` tag and using default `<mat-radio-button>Search</mat-radio-button>` renders slight misalignment of label in UI.

Reply:
Strange, it looks fine in the basic example here: https://material.angular.io/components/radio/overview#radio-overview

Reply:
Exactly, and hence I had added separate `label` tag to avoid this.